lib/src/screens/setup_2fa/setup_2fa_qr_page.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:cake_wallet/core/totp_request_details.dart';
+import 'package:cake_wallet/src/screens/setup_2fa/setup_2fa_enter_code_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+import 'package:cake_wallet/routes.dart';
+import 'package:cake_wallet/src/screens/base_page.dart';
+import 'package:cake_wallet/src/screens/receive/widgets/qr_image.dart';
+import 'package:cake_wallet/utils/show_bar.dart';
+import 'package:cake_wallet/view_model/set_up_2fa_viewmodel.dart';
+
+import '../../../palette.dart';
+import '../../widgets/primary_button.dart';
+import '../../widgets/standard_list.dart';
+
+class Setup2FAQRPage extends BasePage {
+  Setup2FAQRPage({required this.setup2FAViewModel});
+
+  final Setup2FAViewModel setup2FAViewModel;
+
+  @override
+  String get title => S.current.setup_2fa;
+
+  @override
+  Widget body(BuildContext context) {
+  
+    final copyImage = Image.asset(
+      'assets/images/copy_content.png',
+      height: 12,
+      width: 12,
+      color: Color(0xFF355688),
+    );
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 24),
+      child: Column(
+        children: [
+          SizedBox(height: 58),
+          Text(
+            S.current.add_secret_code,
+            style: TextStyle(
+              fontSize: 14,
+              fontWeight: FontWeight.w700,
+              height: 1.5714,
+              color: Palette.darkBlueCraiola,
+            ),
+          ),
+          SizedBox(height: 10),
+          QrImage(
+            size: 327,
+            data: setup2FAViewModel.totpVersionOneLink
```

Comment:
also, I would suggest setting the version for this QrImage to `auto` since the length of this string is variable

Reply:
Alright